Build-your-first-dApp.md

```diff
@@ -202,14 +203,14 @@ Now it's time to create a Solidity smart contract.
 
    &nbsp;
 
-6. Deploy the contract on the Goerli Testnet.
-   - Make sure your Metamask is connected to the Goerli Testnet.
+6. Deploy the contract on the Sepolia Testnet.
+   - Make sure your Metamask is connected to the network where you deployed your contract.
```

Comment:
Maybe we can change this to mention Sepolia or Goerli (or even Mumbai I guess) and maybe mention strongly to make sure Metamask is connected to a test network and not a mainnet. Maybe even upload an image?

Reply:
Can we do this @CJskii?

Reply:
yes, I can update this